scout/test/pageflow/content.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+"""
+Tests the contents of each page, making sure the
+necessary selectors are present
+"""
+
+import bs4
+from django.test import TestCase
+
+
+class ContentTest(TestCase):
+    """Content test set for scout, (page id's)"""
+
+    def returnSoup(self, url):
```

Comment:
getSoup or get_soup would be a better name, also you should implement the thing form the other soup-using tests where it saves each soup in a dictionary so that redundant requests don't get made.